ui/admin/setup-edit-field.php

```diff
@@ -19,7 +19,7 @@
 	}
 }
 
-$ignored_pick_objects = apply_filters( '', array( 'table' ) );
+$ignored_pick_objects = apply_filters( 'pods_ignored_pick_objects', array( 'table' ) );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 0d90e29df3cce4e852bcfd452e58a1ef</sub>